lib/widgets/message_list.dart

```diff
@@ -366,8 +366,11 @@ class MessageListAppBarTitle extends StatelessWidget {
     return Row(
       mainAxisSize: MainAxisSize.min,
       children: [
-        Flexible(child: Text(topic.displayName, style: const TextStyle(
```

Comment:
msglist: Display realmEmptyTopicDisplayName where empty topic appears
    
    Signed-off-by: Zixuan James Li <zixuan@zulip.com>

(Same comment about this being an NFC commit.)